### ISSUE TITLE
Login with Yunohost user details?

### ISSUE THREAD

**Issue by mxmilkiib** [YunoHost-Apps/epicyon_ynh#2]
Trying to login with user details doesn't work.

I'm fairly new to Yunohost, but I guess "Are LDAP and HTTP auth supported? NO" in the README addresses this.

Thought I'd open an issue to track the status of that.

**Comment by gnouts** [comment]
FTR the upstream project does not seem to support LDAP. At least I can not find a mention anywhere.
You should report there in the first place I think ;)

**Comment by anmol26s** [comment]
@mxmilkiib 
You should login with the credentials you entered while installation of the app. The core app do not support LDAP yet.

**Comment by mxmilkiib** [comment]
Very good points.

Have raised upstream https://gitlab.com/bashrc2/epicyon/-/issues/28

**Comment by anmol26s** [comment]
@mxmilkiib 
But you can login as normal user? Have you tried  the credentials set while installation? I am dam sure the account gets created. On my first install I was facing the same problem too.

You can try to change password by:
$ `python3 /var/www/epicyon/epicyon.py --changepassword nickname@domain _newpassword_`
If its not working, please clear the cookies for the url and test it.

**Comment by mxmilkiib** [comment]
I'm sure I set a password but I promptly forgot it and apparently didn't put in my keepass file.

Thanks. (It's in `/opt/yunohost/epicyon`)

Apparently I have also forgotten the username I used, oof.

**Comment by anmol26s** [comment]
Don't worry you will have to wait for the issue to be solved. I tested it yesterday, and login was not working because of some cookie problem I guess. 
I will try to fix it again in next round.
Meanwhile you may try other alternatives.
https://github.com/YunoHost-Apps/zap_ynh
https://github.com/YunoHost-Apps/hubzilla_ynh
https://github.com/YunoHost-Apps/pleroma_ynh
https://github.com/YunoHost-Apps/friendica_ynh
https://github.com/YunoHost-Apps/misskey_ynh
The list is big

**Comment by xavavu** [comment]
Installed Epicyon after recent update. When arriving at initial screen, I'm prompted to enter a Nickname and Password to become site Admin. There is, however, no apparent way to move on from this screen. There is no button to "continue" or"confirm" or "OK"

![ynh_epicyon_first_screen](https://user-images.githubusercontent.com/80236500/126554390-d948e021-2dce-44b1-b03a-c5a3f0e8ad57.jpg)

**Comment by TheNomad11** [comment]
Still the same issue now, no login button

**Comment by p3nj** [comment]
Yeah seems like you have to manually modify [epicyon.service](https://github.com/YunoHost-Apps/epicyon_ynh/blob/8f5e02ad9511c40ad14bc732ad1746b3b2e4f826/conf/systemd.service) file,
change `--registeration close` to `--registration open` in-order to brings back the login button,
but I can't seems login with my yunohost cred too.

**Comment by xavavu** [comment]
@limeless 

> can't seems login with my yunohost cred too.

right... I recall this (LDAP integration) not being supported by the epicyon project.

Thanks for the hint about epicyon.service, will check it out.

**Comment by asherpen** [comment]
Still the same issue now, no login button...
Would someone please fix it?

**Comment by zecamigo** [comment]
> Yeah seems like you have to manually modify [epicyon.service](https://github.com/YunoHost-Apps/epicyon_ynh/blob/8f5e02ad9511c40ad14bc732ad1746b3b2e4f826/conf/systemd.service) file, change `--registeration close` to `--registration open` in-order to brings back the login button, but I can't seems login with my yunohost cred too.

This looks useful

Where is this file located in the system?
I did a quick find and I find several locations, unsure where to apply the changes.

**Comment by p3nj** [comment]
> > Yeah seems like you have to manually modify [epicyon.service](https://github.com/YunoHost-Apps/epicyon_ynh/blob/8f5e02ad9511c40ad14bc732ad1746b3b2e4f826/conf/systemd.service) file, change `--registeration close` to `--registration open` in-order to brings back the login button, but I can't seems login with my yunohost cred too.
> 
> This looks useful
> 
> Where is this file located in the system? I did a quick find and I find several locations, unsure where to apply the changes.

I am not using ynh anymore, you can try to loacte the file inside installation folder of the service, I believe it's somewhere there. If you still having trouble to fine the file try using `find` command: `$ find / -name 'epicyon.service' -type f`.

**Comment by zecamigo** [comment]
> > > Yeah seems like you have to manually modify [epicyon.service](https://github.com/YunoHost-Apps/epicyon_ynh/blob/8f5e02ad9511c40ad14bc732ad1746b3b2e4f826/conf/systemd.service) file, change `--registeration close` to `--registration open` in-order to brings back the login button, but I can't seems login with my yunohost cred too.
> > 
> > 
> > This looks useful
> > Where is this file located in the system? I did a quick find and I find several locations, unsure where to apply the changes.
> 
> I am not using ynh anymore, you can try to loacte the file inside installation folder of the service, I believe it's somewhere there. If you still having trouble to fine the file try using `find` command: `$ find / -name 'epicyon.service' -type f`.

That didn't make the login button appear. Moving on. For reference it's a i386 machine - this seems to make many applications quirky,

**Comment by Graycot** [comment]
I have installed the latest version of epicyon for YUNOHost. 
2022.08.02~ynh1. **The login issue persists.**  After logging in, A large 200 OK message appears.

**Comment by MyNameIsTroll** [comment]
Same problem, I can't connect after installation.

Installed version: 2023.03.17~ynh1



Logs: 
Epicyon service: https://paste.yunohost.org/udarewaqax
Nginx:
https://pastebin.aquilenet.fr/?ff80840c81e82d1e#H8NfX3ecvRQg8a8KCZNMJcE6Ep7UjqWdNkcyx3HvJgsq

:white_check_mark: EDIT
I managed to fix the problem by right-clicking on the login page and displaying the web development tools on firefox, I went to “storage” and deleted the cookies. Then my account that I entered when installing the application works.
